### DISCUSSION TITLE
How do I get all rows after a certain cell (Offset)?

### DISCUSSION THREAD

**Discussion by mayankjha1980** [qax-os/excelize#1482] [Q&A] [answered]
Hi, Let's say I wish to get all rows after cell "A5". The File.GetRows() method will return all rows in the file, but I do not see any methods of functions to get a subset of these rows. Ideally I'm looking for something like [Offset](https://poi.apache.org/apidocs/5.0/org/apache/poi/ss/formula/functions/Offset.html) in ApachePOI.

**Comment by xuri** [comment, marked as answer]
The [streaming reader](https://github.com/xuri/excelize/issues/845#issuecomment-922466641) could skip the specified row's cells when reading the worksheet data. This library doesn't support the formula function OFFSET currently, the formula engine still working in progress, also ref #65.
